config/sync/core.entity_view_display.node.driving_instructor.default.yml

```diff
@@ -31,58 +91,56 @@ content:
     third_party_settings: {  }
   field_di_adi_no:
     type: string
-    weight: 8
+    weight: 2
     region: content
-    label: above
+    label: inline
     settings:
       link_to_entity: false
     third_party_settings: {  }
   field_di_areas:
     type: entity_reference_label
-    weight: 7
+    weight: 6
     region: content
-    label: above
+    label: inline
     settings:
       link: true
     third_party_settings: {  }
   field_di_categories:
-    type: html_list_formatter
-    weight: 6
+    type: entity_reference_label
```

Comment:
Why was this changed from html_list_formatter?

Reply:
That's an oopsy - will change that back

Reply:
No worries... things are changing that quickly with the project it's bound to happen.